projects/packages/publicize/src/rest-api/class-connections-controller.php

```diff
@@ -234,11 +247,15 @@ public static function get_connections( $run_tests = false ) {
 		}
 
 		$path = add_query_arg(
-			array( 'test_connections' => $run_tests ),
+			$args,
```

Comment:
Should we move the logic to only pass `test_connections` here, rather than in get_items?
https://github.com/Automattic/jetpack/pull/40704/files#diff-0dd1f075518bd69a442b383d510c057910f57b439128159e7a2a54d8c1182d3dR282-R285

It won't matter as we'll ignore it, but when we call this function with `scope` in the args, it will get added to the URL.

Reply:
Or possibly have it in both places?